src/App.css

```diff
@@ -89,10 +107,22 @@ li {
 }
 
 .input {
-  width: 25rem;
+  width: 27rem;
   margin: 1rem;
 }
 
+.book-tools {
+  display: flex;
+}
+
+.book-tools :nth-child(2) {
+  border-left: 1px solid #dcdcdc;
+  border-right: 1px solid #dcdcdc;
+  padding: 0 1rem;
+}
+
```

Comment:
- [ ] Good job so far 👏 You have made a great effort to implement the suggested changes by the previous reviewers. However, from the project requirements, you are expected to implement the design given precisely as it is. There is a simple change that has been suggested by the previous reviewers repeatedly but you have not implemented one of them. Pleas look at the following comparison 👇 
**Given design**
![image](https://user-images.githubusercontent.com/79058364/191857667-6f501420-6d4a-41ab-a615-f0829889a246.png)

**Your design**
![image](https://user-images.githubusercontent.com/90222110/209431816-41370ea6-5401-429a-81e4-e905ced6447f.png)